BodoSQL/calcite_sql/bodosql-calcite-application/src/main/java/com/bodosql/calcite/application/RelationalAlgebraGenerator.java

```diff
@@ -424,7 +423,7 @@ public void resetPlanner() {
    * @param sql Query to parse Sets parseNode to the generated SQLNode
    * @throws SqlSyntaxException if the SQL syntax is incorrect.
    */
-  public void parseQuery(String sql) throws SqlSyntaxException {
+  void parseQuery(String sql) throws SqlSyntaxException {
```

Comment:
These APIs can be moved to package private. While any code can use the Python APIs, this helps clarified that they are not the desired API for the Java code.